lib/cms_tags/filter_checkboxes.rb

```diff
@@ -1,57 +1,39 @@
 class FilterCheckboxes < ComfortableMexicanSofa::Content::Tag::Fragment
   def initialize(context:, params: [], source: nil)
     super
-    @custom_categories = options['options'].split(',').map do |category|
-      category = category.strip
-    end
-
-    @custom_categories.each do |category|
-      class_eval { attr_accessor category }
-      instance_variable_set "@#{category}", false
-    end
-  end
-
-  def store_in_content(category)
-    value = self.send(category)
-    fragment.content << " #{category} #{value} "
+    @custom_categories = options['options'].split(',').map(&:strip)
   end
   
   def form_field(object_name, view, index)
     name = "#{object_name}[fragments_attributes][#{index}][content]"
```

Comment:
Here's what I don't really understand about the variables inside the square brackets - they're not indices of any array, the square brackets seem to mean something else in this context. Although when I changed `boolean` to `content` in the last bracket, I could tell that it's basically specifying which attribute of the current tag to use.

Reply:
`fragments_attributes` it's not about array indices, it's basically related to the nested object attributes.
For example, here we have a form for a Page object, to assign values to it's nested object, in this case a Fragment, you can use the above syntax